Makefile

```diff
@@ -72,6 +72,10 @@ lint:
 postgresql postgres:
 	make -C local-data-warehouses postgresql
 
+.PHONY: trino
+trino:
+	docker run --name trino -d -p 8080:8080 trinodb/trino
```

Comment:
Note: this fails if you've already run it once and haven't removed the existing installation. Or at least that happened to me when I tried to add this (long story, but note `trinodb/trino` and `trinodb/trinodb` are not the same thing and it took me like an hour to figure that one out).

Anyway, maybe it's worth adding a comment here to indicate that you can just start your existing docker hub install if you already have this working locally with the `trino` name.

I don't think Postgres has the same issue because of the docker file but I admit I can't be bothered to try it.

Reply:
Ended up adding the docker-compose file and its working!